app/index.html

```diff
@@ -13,8 +12,7 @@
 		<meta name="theme-color" content="#5d1ae6" />
 		<meta name="og:image" content="https://radio4000.com/favicon.png" />
 		{{content-for "head"}}
-		<link rel="preload" as="font" href="https://res.cloudinary.com/radio4000/raw/upload/v1492541388/maisonneueweb-book_ee98sm.woff2" type="font/woff2" crossorigin />
-		<link rel="preload" as="font" href="https://res.cloudinary.com/radio4000/raw/upload/v1492541388/maisonneueweb-bold_rvmbzr.woff2" type="font/woff2" crossorigin />
+		<link rel="preconnect" href="https://res.cloudinary.com" />
```

Comment:
Why is there no font referenced in the new line?

Reply:
Good question.  Not sure.

In theory it should be faster to preload the fonts, but since we now have 1) service worker 2) preconnect for the same res.cloudinary.com domain, I thought simpler to remove...

Reply:
in my very unscientific tests adding the preconnect saves ~4-500ms here.